src/reim/autoPhaseCorrection.js

```diff
@@ -74,8 +69,8 @@ export function autoPhaseCorrection(data, options = {}) {
 function autoPhaseRegion(re, im, x0) {
   let start = -180;
   let stop = 180;
-  let nSteps = 20;
-  let maxSteps = 3;
+  let nSteps = 2;
+  let maxSteps = 13;
```

Comment:
Should not those 2 parameters be options if you need to change them from time to time ?

Reply:
I am just looking for a better sampling